src/ImageIO.jl

```diff
@@ -103,6 +104,16 @@ function save(s::Stream{DataFormat{:TIFF}}, image::S; permute_horizontal=false,
     end
 end
 
+## OpenEXR
+
+function load(f::File{DataFormat{:EXR}})
+    Base.invokelatest(checked_import(idOpenEXR).load, f)
+end
```

Comment:
1. We need to add a `canonical_type` for EXR format and make sure the return type is `Array` or whatever standard format (see #26)
2. for future extensibility, we can unconditionally pass whatever keyword arguments to `load` and `save`.

```suggestion
function load(f::File{DataFormat{:EXR}}; kwargs...)
    data = Base.invokelatest(checked_import(idOpenEXR).load, f, kwargs...)
    return convert(canonical_type(s, data), data)
end
```

Reply:
Done.